packages/rocketchat-api/server/api.js

```diff
@@ -178,17 +178,26 @@ const getUserAuth = function _getUserAuth() {
 	};
 };
 
-RocketChat.API.v1 = new API({
-	version: 'v1',
-	useDefaultAuth: true,
-	prettyJson: true,
-	enableCors: false,
-	auth: getUserAuth()
-});
+let enableCors = RocketChat.settings.get('API_Enable_CORS');
+
+const createApi = function() {
+	RocketChat.API.v1 = new API({
+		version: 'v1',
+		useDefaultAuth: true,
+		prettyJson: true,
+		enableCors,
+		auth: getUserAuth()
+	});
+
+	RocketChat.API.default = new API({
+		useDefaultAuth: true,
+		prettyJson: true,
+		enableCors,
+		auth: getUserAuth()
+	});
+};
 
-RocketChat.API.default = new API({
-	useDefaultAuth: true,
-	prettyJson: true,
-	enableCors: false,
-	auth: getUserAuth()
+RocketChat.settings.onload('API_Enable_CORS', (key, value) => {
+	enableCors = value;
+	createApi();
```

Comment:
@graywolf336 that's what I meant writing y'days comment. 
Is that the proper way to do it?

_Note to myself: I really should get an IDE on my mobile..._